packages/node/package.json

```diff
@@ -77,7 +81,7 @@
     "preset": "ts-jest",
     "testEnvironment": "node",
     "moduleNameMapper": {
-      "axios": "axios/dist/node/axios.cjs"
+      "^axios$": "axios/dist/node/axios.cjs"
```

Comment:
without that change, tests for retries don't work.

I observed strange behavior - it sends GET request to `127.0.0.1:80` even when you send POST request to baseUrl, it messes up the request config.

change to `^axios$` fixed that issue. This is also recommendation inside [nock readme for axios](https://github.com/nock/nock#axios).

I also tested manually using symlink in testing repo's `node_modules` and retries work as expected.